README.md

```diff
@@ -121,17 +121,100 @@ Resolved environments can also be created via the API:
 
 ## Quickstart
 
-First, install Rez. Download the source, and from the source directory, run
-(with DEST_DIR replaced with your install location):
+Choose your installation method.
 
-    ]$ python ./install.py -v DEST_DIR
+<details>
+  <summary>Simple</summary>
+<table>
+<tr><td>
+<br>
+
+**Simple** The simple approach is well suited for beginners and those looking to learn more about Rez and whether it is suited to their problem and environment.
+
+</td></tr>
+<tr><td>
+
+```bash
+$ pip install rez
+```
+</td></tr>
+<tr><td>
+
+<br>
+
+**Advantages**
+
+- **User-friendly and familiar installation method** Everybody loves pip
+
+</td></tr>
+</table>
+
+</details>
+</details><details>
+  <summary>Recommended</summary>
+
+<table>
+<tr><td>
+<br>
+
+**Recommended** The recommended approach is better suited for live production, where you've committed to Rez and want precision and control.
+
+</td></tr>
+<tr><td>
+
+```bash
+$ python -m virtualenv rez
+$ rez\Scripts\activate
+(rez) $ pip install rez
+```
+</td></tr>
+<tr><td>
+<br>
+
+**Advantages**
+
+- **Isolated PATH** No interference from external binaries in your `Scripts/` Python directory.
+- **Isolated site-packages** No interference from system-wide Python packages
+
+</td></tr>
+</table>
+</details>
+<details>
+  <summary>Developer</summary>
+
+<table>
+<tr><td>
+<br>
+
+**Developer** The developer approach maintains Git history and enables you to contribute back to this project (yay!)
+
+</td></tr>
+<tr><td>
+
+```bash
+$ python -m virtualenv rez-dev
+$ rez-dev\Scripts\activate
+(rez) $ git clone https://github.com/nerdvegas/rez.git
+(rez) $ cd rez
+(rez) $ pip install . -U
```

Comment:
The real development mode for a pip package is to use `pip install . -e`
```
  -e, --editable <path/url>   Install a project in editable mode (i.e. setuptools "develop mode") from a local project path or a VCS url.
```
That symlinks your code in the site-packages directory. That way you do it only once and your code is kept up-to-date in your site-packages.